test/data/getAccount_0x0000000000000000000000000000000000001438.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Made this a symlink (to the already existing account) to be explicit about what was added and avoid duplication.